compiler/p/codegen/OMRTreeEvaluator.cpp

```diff
@@ -6220,3 +6220,41 @@ TR::Register *OMR::Power::TreeEvaluator::retrieveTOCRegister(TR::Node *node, TR:
 {
    return cg->machine()->getPPCRealRegister(TR::RealRegister::gr2);
 }
+
+TR::Register * OMR::Power::TreeEvaluator::ibyteswapEvaluator(TR::Node *node, TR::CodeGenerator *cg)
+{
+   TR_ASSERT(node->getNumChildren() == 1, "Wrong number of children in inlineIntegerReverseBytes");
+
+   TR::Node *firstChild = node->getFirstChild();
+   TR::Register *tgtRegister = cg->allocateRegister();
+
+   if (TR::Compiler->target.cpu.id() >= TR_PPCp7 &&
```

Comment:
I know the code you're refactoring had this constraint, but I don't think we need to limit this to >=P7, support for this instruction goes back a long way. @zl-wang, did we limit this to P7 and later for any particular reason?